tests/system_tests/test_core/test_public_api.py

```diff
@@ -321,17 +321,21 @@ def test_run_create(user, relay_server):
 
 
 def test_run_update(user, relay_server):
-    seed_run = Api().create_run()
+    seed_run = wandb.init(config={"foo": "not_bar"})
```

Comment:
We prefer the wandb_init fixture, could you try this and see if it works?

```suggestion
def test_run_update(wandb_init, relay_server):
    seed_run = wandb_init(config={"foo": "not_bar"})
```

Reply:
Done! thanks for the suggestion.